gaen-processes.md

```diff
@@ -0,0 +1,40 @@
+# Exposire Notification Processes
+
+This document describes the processes, related to GAEN, used in the app to perform distributed contact registration, letting an infected user notify their digital contacts, and notifying app users of exposure.
```

Comment:
`registration` not `registration`

Reply:
Correction an actual content the same here?